eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -3,68 +3,81 @@
 $GithubAPIBaseURI = "https://api.github.com/repos"
 
 function Get-GitHubApiHeaders ($token) {
-  $headers = @{
-    Authorization = "bearer $token"
-  }
-  return $headers
+    $headers = @{
```

Comment:
Remove the indentation changes and this file change must be made on azure-sdk-tools repo.